backend/spec/requests/api/portfolios_spec.rb

```
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe "Api::PortfoliosController", type: :request do
+  describe "GET /api/portfolios" do
+    let!(:user) {
```

rubocop (1.69.1)

[convention] Avoid using `{...}` for multi-line blocks.

(Style/BlockDelimiters)